Lib/test/test_poll.py

```diff
@@ -130,7 +130,7 @@ def test_poll2(self):
         p = proc.stdout
         pollster = select.poll()
         pollster.register( p, select.POLLIN )
-        for tout in (0, 1000, 2000, 4000, 8000, 16000) + (-1,)*10:
+        for tout in (0, 500, 1000, 2000, 4000, 8000, 16000, None, -1000) + (-1,)*8:
```

Comment:
#4003 is merged. Please revert this change and add the timeout value -1000 in the new test method.

Reply:
@serhiy-storchaka sure, will do.
What is the preferred method to update this branch with the new upstream code, `git merge`? I'm asking because this is my first patch and the developer guide mention to not use _anything that would require a force push to GitHub_, so I guess a rebase is excluded.

Reply:
Yes, just use `git merge`.